### PR TITLE
arch/arm: Move FPU initialization to common place

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Impact
Minor, code refactor

## Testing
Pass CI

### CODE REVIEW

File: arch/arm/src/eoss3/eoss3_start.c [hunk 1]
```diff
@@ -221,7 +122,7 @@ void __start(void)
 
   /* Unclear what needs to happen here to make RENODE happy. */
 #if 1
```
Comment (pkarashchenko) [inline arch/arm/src/eoss3/eoss3_start.c:124]:
```suggestion
```

File: arch/arm/src/eoss3/eoss3_start.c [hunk 2]
```diff
@@ -221,7 +122,7 @@ void __start(void)
...
-  eoss3_fpuconfig();
+  arm_fpuconfig();
 #endif
```
Comment (pkarashchenko) [inline arch/arm/src/eoss3/eoss3_start.c:126]:
```suggestion
```

**Review by pkarashchenko — APPROVED** [review]

File: arch/arm/src/eoss3/eoss3_start.c (hunk 1 above)
Comment (xiaoxiang781216) [reply arch/arm/src/eoss3/eoss3_start.c]: Done.

File: arch/arm/src/eoss3/eoss3_start.c (hunk 2 above)
Comment (xiaoxiang781216) [reply arch/arm/src/eoss3/eoss3_start.c]: Done.

**Pull request merged**